packages/api/src/models/Payment.ts

```diff
@@ -11,6 +11,20 @@ export enum PaymentType {
 	Check = 'Check',
 	Credit = 'Credit',
 	Cash = 'Cash',
+	BankTransfer = 'Bank Transfer',
+	CreditCard = 'Credit Card',
+	Debit = 'Debit',
+	PayPal = 'PayPal',
+	TwoCheckout = '2Checkout',
+	Visa = 'VISA',
+	Mastercard = 'MASTERCARD',
+	Discover = 'DISCOVER',
+	Nova = 'Nova',
+	Amex = 'AMEX',
+	Diners = 'DINERS',
+	Eurocard = 'EUROCARD',
+	Jcb = 'JCB',
+	Ach = 'ACH',
```

Comment:
We also support "Other": https://github.com/freshbooks/evolve/blob/main/evolve/service/payment/payment_validate.py#L34